src/main/java/com/m4thg33k/tombmanygraves/events/CommonEvents.java

```diff
@@ -229,10 +230,15 @@ private BlockPos checkLevel(World world, BlockPos pos, int radius, boolean ignor
 
     private boolean isValidLocation(World world, BlockPos pos, boolean ignoreFluidConfigs)
     {
-        if (pos.getY() < 0 || pos.getY() >= world.getActualHeight())
+        if ((pos.getY() < 0 || pos.getY() >= world.getActualHeight()) && !Loader.isModLoaded("cubicchunks"))
         {
             return false;
         }
+        if (pos.getY() < world.getMinHeight() || pos.getY() >= world.getMaxHeight())
```

Comment:
That isn't going to compile without including cubic chunks as dependency. Use `World.isOutsideBuildHeight` instead.

Reply:
Wouldn’t adding
```java
Loader.isModLoaded("cubicchunks") && 
```
To the beginning also work? (Just wondering)

Reply:
No because the compiler doesn't know that cubic chunks exists. You clearly didn't actually test the code. Also with `isOutsideBuildHeight` the first check for height is completely unnecessary.

Reply:
“I currently don't have the material to actually test it in game.” so indeed, I didn’t test it.
Ah yes, ofc it doesn’t know. Sorry, that was just a dumb moment on my side.
And ok, I’ll fix that.